prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -776,6 +777,9 @@ abstract class CoolTest {
         val qualityReversedReceiver = settings.receivers.filter {
             it.organizationName == orgName && it.name == "QUALITY_REVERSED"
         }[0]
+        val settingsTestReceiver = settings.receivers.filter {
+            it.organizationName == orgName && it.name == "SETTINGS_TEST"
+        }[0]
```

Comment:
This can be replaced by a call to `first` instead of using the subscript:

```
settings.receivers.first { it.organizationName == orgName && it.name == "SETTINGS_TEST" }
```